<launch>
    <node name="cv_ad" pkg="rm_cv" type="armor_detection_node" cwd="ROS_HOME" launch-prefix="sudo -E bash -c" output="screen">
        <param name="yaw_kp" type="double " value="0.1"/>
        <param name="yaw_ki" type="double " value="0"/>
        <param name="yaw_kd" type="double " value="0"/>
        <param name="pitch_kp" type="double " value="0.1"/>
        <param name="pitch_ki" type="double " value="0"/>
        <param name="pitch_kd" type="double " value="0"/>
        <param name="maxPitchVelocity" type="double " value="1.0"/>
        <param name="maxYawVelocity" type="double " value="1.0"/>
        <param name="GimbalControllerTimeout" type="double " value=" 0.2"/>
    </node>
</launch>
